packages/pds/pds.6.55/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+maintainer: "orbitz@gmail.com"
+build: [
+	[make "-j%{jobs}%"]
+	[make "-j%{jobs}%" "test"] {with-test}
+]
+
+install: [
+	[make "PREFIX=%{prefix}%" "install"]
+]
+
+remove: [
+	[make "PREFIX=%{prefix}%" "remove"]
+]
+
+depends: [
+	"cmdliner" { >= "1.3.0" }
+	"containers" { >= "3.12" }
+	"crunch"
+	"fmt"
+	"logs"
+	"ocaml" { >= "4.13.0" }
```

Comment:
```suggestion
	"ocaml" { >= "5.1.0" }
```
Due to the use of [String_set.to_list)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/b68321e88aff29f87b477392376f999d1effcc57/variant/compilers,5.0,pds.6.55)